README.md

```diff
@@ -1,2 +1,57 @@
-# will-legislate-for-money
-Django application for viewing congressperson contributions and votes
+**About Will Legislate For Money**
+
+Will Legislate For Money is a Python/Django app built as a tool to give voters easier access to information on their legislators’ campaign funding and congressional action.
+
+![alt text](https://cloud.githubusercontent.com/assets/1721599/26426097/6a793574-4094-11e7-809f-993d644a85b0.png)
+
+You can use the live site at http://legislate.money
+
+**Setup**
+
+If you are new to Python/Django, you can use this guide to help set up your environment:
+
+    http://www.obeythetestinggoat.com/book/pre-requisite-installations.html
+
+Will Legislate For Money uses Python 3 and Django 1.11rc1.
+
+Clone down this repo.
+
+Dependencies:
+
+In your virtual environment, install:
+
+    requests
+    
+    shutil
+
+To set up the database and seed data:
+
+    python manage.py migrate
+    python manage.py seed_legislators
```

Comment:
Also need `python manage.py get_images`

other note: the images need to be added to the .gitignore. I'll try to include that in my next PR